header: updated quick actions dropdown.

alejandromumo opened the pull request:
closes https://github.com/zenodo/rdm-project/issues/486

alejandromumo commented:
Result:

## Unauthenticated users

![Screenshot at Nov 01 14-06-59](https://github.com/inveniosoftware/invenio-app-rdm/assets/21204744/04dc5399-faa8-4c57-a968-213468d29363)

If the user clicks "New upload" or "New community", they're redirected to the login page

## Authenticated users

![Screenshot at Nov 01 14-06-49](https://github.com/inveniosoftware/invenio-app-rdm/assets/21204744/71d808d0-5896-45ff-9895-799bac6893a5)

lnielsen commented:
Can you keep the original menu item text:

- New upload
- New community

Also the caret down icon is missing after the ``...``

alejandromumo commented:
> Can you keep the original menu item text:
> 
>     * New upload
> 
>     * New community
> 
> 
> Also the caret down icon is missing after the `...`



![Screenshot at Nov 02 10-04-29](https://github.com/inveniosoftware/invenio-app-rdm/assets/21204744/34dee8b5-0df6-4bd6-8f28-e1b57f539417)

kpsherva commented:
small suggestion: could you increase a distance between + and New ? it seems a little bit too cramped together

alejandromumo commented:
> small suggestion: could you increase a distance between + and New ? it seems a little bit too cramped together

I am not sure if what I did is correct in terms of good practices (let me know if it isn't): I added an ID to the label and added a margin-left. 

<img width="1295" alt="Screenshot at Nov 06 09-41-55" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/21204744/d16b31b8-e71c-4824-b3ae-c111713ad207">

ntarocco commented:
I am sure you did, but just to be sure, does the link correctly redirect to login when anonymous?

jbenito3 commented:
> > small suggestion: could you increase a distance between + and New ? it seems a little bit too cramped together
> 
> I am not sure if what I did is correct in terms of good practices (let me know if it isn't): I added an ID to the label and added a margin-left.
> 
> <img alt="Screenshot at Nov 06 09-41-55" width="1295" src="https://user-images.githubusercontent.com/21204744/280643741-d16b31b8-e71c-4824-b3ae-c111713ad207.png">

it seems to me like now there is a huge space on the left...

kpsherva commented:
after some discussions, we decide it to postpone it until optimal solution is found

carlinmack commented:
Closed via https://github.com/inveniosoftware/invenio-communities/pull/1065
